doc/rst/platforms/udp.rst

```diff
@@ -153,9 +153,10 @@ I get xSocket errors when using a system with multiple IP addresses
  *** FATAL ERROR: Got an xSocket while spawning slave process: connect()
  failed while creating a connect socket (111:Connection refused)
 
-You need to set ``GASNET_MASTERIP`` and possibly ``GASNET_WORKERIP``.
-Please refer to:
+You need to set ``CHPL_LAUNCH_MASTERIP`` (or ``GASNET_MASTERIP``), and possibly
```

Comment:
I don't know of any other env vars with a `CHPL_LAUNCH_` prefix. Should it be `CHPL_LAUNCHER_` (or `CHPL_RT_`) instead?

Reply:
I would be okay with changing the name to either of those options.  Do you or @gbtitus have a strong preference?

Left to my own devices, I would go with `CHPL_LAUNCHER_`

Reply:
Of those two I prefer `CHPL_RT_...`.  I believe the "RT" in `CHPL_RT_...` means that the given variable has its effect at run (i.e. execution) time, rather than say at compile time.  I.e., it doesn't necessarily mean that the variable is interpreted by the Chapel runtime proper.  And I'd prefer we not assert or imply that some particular part of the Chapel stack is the thing that's associated with a given env var, since that might change, or not be universally true.  So I'm not fond of `CHPL_LAUNCHER_...` or `CHPL_COMPILER_...` or `CHPL_MODULES_...` etc.